### DISCUSSION TITLE
求教一个给参考文献加编号问题

### DISCUSSION THREAD

**Discussion by giserzy** [whutug/whu-thesis#162] [Q&A] [answered]
求教模板reference增加[1],[2],[3]这种数字索引。

**Comment by tanukihee** [comment, marked as answer]
请参照使用文档

**Reply by giserzy** [reply]
谢谢!
给大家留下参考方法
住tex文件下bib-style的设置
style              =
      {
        graphics-path  = {{figures/}{data/}},
        list-of-figures,
        list-of-tables,
        bib-backend={bibtex},
        bib-style={author-year-numbered}
      },
